### DISCUSSION TITLE
tftp-hpa package misleading description

### DISCUSSION THREAD

**Discussion by DeJQit** [msys2/MSYS2-packages#2655] [General]
Why package description is "Official tftp server" but actually server part removed from package?

https://github.com/msys2/MSYS2-packages/blob/fd33629ef88f656b60f482bccef1e629568a6faf/tftp-hpa/PKGBUILD#L75

**Comment by lazka** [comment]
Unclear from the git history. Maybe it didn't work. Feel free to give it a try.

**Reply by DeJQit** [reply]
It doesn't work. I cannot find the tftpd.exe  file in the repository.
![image](https://user-images.githubusercontent.com/3215127/141269656-2be2f0bc-708f-4c69-b11d-52c2a44829ac.png)

**Reply by lazka** [reply]
I meant re-building the package with those binaries and testing then :)

**Reply by DeJQit** [reply]
> 
> 
> I meant re-building the package with those binaries and testing then :)

Oh, well. I'll try build and test. I'll write what happens. Thanks)
